### DISCUSSION TITLE
Sponsored Brand product sales and spend

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/ads-advanced-tools-docs#195] [Q&A]
Hi,

Does anyone know how to get spend and sales of Sponsored Brand and Sponsored Brand Video at ASIN or SKU level? Is there any API to get this?

@elizafarley Can you please help with this?

**Comment by drewxa** [comment]
Explore the possibilities of the Report API, as outlined in the documentation: https://advertising.amazon.com/API/docs/en-us/guides/reporting/v3/report-types/overview. For your specific needs, I recommend looking into the **Ad** or **Purchased Product** types of reports. If I were in your position, I would conduct thorough research in this direction.
